charts/azure-managed-cluster/values.yaml

```diff
@@ -0,0 +1,144 @@
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+# This follow a aks production economy configuration
+
+## AKS cluster configuration
+
+## The subscription ID
+subscriptionID: ''
+identity:
+  clientID: ''
+  tenantID: ''
+  clusterIdentityName: cluster-identity
+  clusterIdentitySecretName: cluster-identity-secret
+  clusterIdentitySecretNamespace: default
+  resourceID: ""
+  type: UserAssignedMSI
+
+cluster:
+  ## The resource group for the AKS cluster object
+  resourceGroupName: aksclusters
+
+  ## The resource group name for the nodes
+  nodeResourceGroupName: base
+
+  ## The location of the cluster
+  location: eastus
+
+  ## The name of the cluster
+  name: aks-cluster-api
+
+  ## the subnet 
+  cidrBlocks:
+  - 192.168.0.0/16
+
+  ## The sku of the control plane
+  sku: Standard
+
+
+controlplane:
+
+  ## SSH public key (must be valid)
+  
+  ## Kubernetes version
+  kubernetes_version: "v1.29.2"
+
+  ## Network policy, "calico" (for kubenet and azurecni) or "azure" (for azureCNI)
+  networkPolicy: "calico"
+
+  ## Network Plugin "kubenet" or "azure"
+  networkPlugin: "azure"
+  networkPluginMode: "overlay"
+  sshPublicKey: ""
+  outboundType: "loadBalancer"
+  dnsServiceIP:  ""
+  loadBalancerSKU: "Standard"
+  sku:
+    tier: Standard
+  addonProfiles : 
+    - name: azurepolicy
+      enabled: false
+      config: {}
+    - name: httpApplicationRouting
+      enabled: false
+      config: {}
+    - name: virtual-node
+      enabled: true
+      config: {}
+  loadBalancerProfile:
+    managedOutboundIPs: 1
+    outboundIPs: ""
+    outboundIPPrefixes: ""
+    allocatedOutboundPorts: 0
+    idleTimeoutInMinutes: 30
+  apiServerAccessProfile: 
+    authorizedIPRanges: []
+    spec: 
+      enablePrivateCluster: false
+      privateDNSZone: ""
+      enablePrivateClusterPublicFQDN: false
+  autoscalerProfile: 
+    balanceSimilarNodeGroups: "false"
+    expander: "random"
+    maxEmptyBulkDelete: "10"
+    maxGracefulTerminationSec: "600"
+    maxNodeProvisionTime: "15m"
+    maxTotalUnreadyPercentage: "45"
+    newPodScaleUpDelay: "0s"
+    okTotalUnreadyCount: "3"
+    scanInterval: "10s"
+    scaleDownDelayAfterAdd: "10m"
+    scaleDownDelayAfterDelete: "0s"
+    scaleDownDelayAfterFailure: "3m"
+    scaleDownUnneededTime: "10m"
+    scaleDownUnreadyTime: "20m"
+    scaleDownUtilizationThreshold: "0.5"
+    skipNodesWithLocalStorage: "false"
+    skipNodesWithSystemPods: "false"
+  azureEnvironment: "AzurePublicCloud"
+  kubeletUserAssignedIdentity: ""
+  oidcIssuerProfile: ""
+  # disableLocalAccounts: false
+  additionalTags:
+     - key: test
+       value: test
+  dnsPrefix : ""
+  virtualNetwork: {}
+  disableLocalAccounts: false
+#Only 1 Nodepool of mode System is allowed  
+agentpools: 
+  agentpool0:
+    mode: System
+    sku: Standard_DS2_v2
+    nodecount: 5
+    osDiskSizeGB: 100
+    additionalTags:
+     - key: test
+       value: test
+   #  availabilityZones: 
```

Comment:
If we intend to leave this section in as an example of how to enable Availability Zones, let's add a comment to that effect.